### DISCUSSION TITLE
Parcel - image src picsum photos not working ?

### DISCUSSION THREAD

**Discussion by shulinnn** [parcel-bundler/parcel#8644] [Q&A] [answered]
Hi , sorry for the bad title , im kinda lost. My first time using Parcel, have just pure vanilla parcel config without plugins.
Just wondering what could be wrong with parcel ? The first image shows as expected but the others have just "error icon" Even tho in  the network tab the one which is workign had 302 warning and the not working ones have 300 status code.
This happened with every placeholder image tool i tried, so i guess its something with Parcel config, any help would be appreciated

```html
        <main class="container">
            <div class="flex flex-col w-full gap-4">
                <div class="flex w-full">
                    <img src="http://picsum.photos/1037/654" />
                </div>
                <div class="flex flex-row w-full justify-between">
                    <img class="w-1/6 h-auto" src="http://picsum.photos/150/150" />
                    <img class="w-1/6 h-auto" src="http://picsum.photos/150/150" />
                    <img class="w-1/6 h-auto" src="http://picsum.photos/150/150" />
                    <img class="w-1/6 h-auto" src="http://picsum.photos/150/150" />
                    <img class="w-1/6 h-auto" src="http://picsum.photos/150/150" />
                </div>
            </div>
        </main>
```

Also in the elements tab the one which is working is looking like this 
`<img src="http://picsum.photos/1037/654" />`

While the ones not working are like this.

`<img class="w-1/6 h-auto" src="f66f24f05cdd5a3a">`

**Comment by mischnic** [comment]
What Parcel version are you using? For me, all URLs are correct in the output

**Comment by shulinnn** [comment, marked as answer]
`"parcel": "^2.8.0"`
Also , i got it temporarily fixed (even tho i dont understand it a bit) by changing protocol in the lorem picsum url to https -> saving and then changing back to http.
Literally no clue why it works now.
